src/illuminatio/rule.py

```diff
@@ -45,10 +60,16 @@ def __repr__(self):
         return self.__str__()
 
     def to_dict(self) -> dict:
+        """
+        converts a rule into a dictionary
+        """
         return {AFFECTED_POD_IDENTIFIER: self.concerns, "allowed": list(map(dict_mapper, self.allowed))}
 
     @classmethod
     def from_network_policy(cls, net_pol: k8s.client.V1NetworkPolicy):
+        """
+        returns a class containing the concerns and rules of a given NetworkPolicy
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)